python/ray/data/context.py

```diff
@@ -14,6 +15,9 @@
 # Whether block splitting is on by default
 DEFAULT_BLOCK_SPLITTING_ENABLED = False
 
+# Whether pandas block format is enabled.
+DEFAULT_ENABLE_PANDAS_BLOCK = env_bool("RAY_DATA_PANDAS_BLOCK_FORMAT", True)
```

Comment:
Please remove the env bool here, I don't think we want to support env variables as config for datasets, as that harms reproducibility.

Reply:
Done in another PR.